cli/spark/endpoints/cluster_copy.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import typing
+from cli import log
+from cli import utils
+from cli.spark.aztklib import load_spark_client
+import aztk.spark
+
+
+def setup_parser(parser: argparse.ArgumentParser):
+    parser.add_argument('--id', dest='cluster_id', required=True,
+                        help='The unique id of your spark cluster')
+
+    parser.add_argument('--source-path', required=True,
```

Comment:
can/should this include directories? (happy to push that out to later)

Reply:
yeah -- it is just an sftp client under the hood so directories will work fine.